docs/howto/backup-and-restore.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Backup and Restore Repository
+description: Use the lakeFS DumpRefs and RestoreRefs commands to backup and restore lakeFS repository
+parent: How-To
+---
+
+# Backup and Restore Repository
+This section explains how to backup and restore lakeFS repository for different use-cases:
+1. Disaster Recovery: you want to backup the repository regurlarly so you can restore it in case of any disaster.
```

Comment:
```suggestion
1. Disaster Recovery: you want to backup the repository regularly so you can restore it in case of any disaster. You'd also need to make sure to copy the repository's storage namespace to another, preferably geographically separate location after each successful dump.
```